### PR TITLE
Change pwd2

### PR DESCRIPTION
**Pull request by pwalczysko**
This PR is trying to make use of the logic specified in [Simon's comment](https://github.com/openmicroscopy/prod-playbooks/pull/76#discussion_r195341856). 

Also, it fixes the typo in the override variable name. 

cc @joshmoore @manics @kennethgillen

### CODE REVIEW

File: omero/training-server/playbook.yml [hunk 1]
```diff
@@ -292,7 +292,10 @@
     omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
     omero_cli_duplicate_release: "{{ omero_cli_duplicate_release_override | default('0.2.0') }}"
     omero_cli_render_release: "{{ omero_cli_render_release_override | default('0.3.0') }}"
-    os_system_users_password: "{{ os_systems_users_password_override | default('$6$sjH0EeRKZdWYc1y8$F79INhPIfwVUfZ/DRrk.rYEetacoxoqxMVSVRavjggro84vpCqcxUhcNs79C5y4RmdLBb/3sA3vRagfNVjMxD.') }}"
+    # The os_system_users_password default is "ome".
+    # You might wish to override it by a private variable on your system,
```
Comment (manics) [inline omero/training-server/playbook.yml]: Would you mind changing the wording to something like "You may wish to change this variable, or override it with a private variable os_system_users_password_override"

**Comment by pwalczysko** [comment]
@manics See https://github.com/openmicroscopy/prod-playbooks/pull/81/commits/c8cda1f2e785e1a61a613af20eaf35e722d27f88, is that any better ?

File: omero/training-server/playbook.yml [hunk 2]
```diff
@@ -292,7 +292,10 @@
     omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
     omero_cli_duplicate_release: "{{ omero_cli_duplicate_release_override | default('0.2.0') }}"
     omero_cli_render_release: "{{ omero_cli_render_release_override | default('0.3.0') }}"
-    os_system_users_password: "{{ os_systems_users_password_override | default('$6$sjH0EeRKZdWYc1y8$F79INhPIfwVUfZ/DRrk.rYEetacoxoqxMVSVRavjggro84vpCqcxUhcNs79C5y4RmdLBb/3sA3vRagfNVjMxD.') }}"
+    # The os_system_users_password default is "ome".
+    # You may wish to change this variable,
+    # or override it with a private variable os_system_users_password_override.
```
Comment (kennethgillen) [inline omero/training-server/playbook.yml]: @pwalczysko - "or override it by defining the private variable os_system_users_password_override" ?

**Comment by pwalczysko** [comment]
@kennethgillen pushed

**Comment by kennethgillen** [comment]
LGTM too 👍

**Pull request merged**